src/pe_reports/data/data_schema.sql

```diff
@@ -558,6 +558,33 @@ CREATE TABLE public.unique_software (
     software_name text NOT NULL
 );
 
+--
+--Name: was_tracker_customerdata
```

Comment:
This should match the other tables:
```suggestion
-- Name: was_tracker_customerdata; Type: TABLE; Schema: public; Owner: pe
```